application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -76,6 +76,8 @@ adminTools.dashboard.healthcheck.database.info=Database status OK.
 adminTools.dashboard.healthcheck.database.warn=Database not found!
 adminTools.dashboard.healthcheck.database.notSupported=The currently used database is {0}, which is not supported by XWiki. Check the help links to find the database compatibility requirements.
 adminTools.dashboard.healthcheck.description=Check if your instance is configured correctly and is properly running. For any issues, check out this {0}.
+adminTools.dashboard.healthcheck.flushCache=Flush Wiki cache
```

Comment:
`Flush instance cache`
And you can reuse it for the modal title, since it's the same text